src/components/organize/tasks/TaskList/index.tsx

```diff
@@ -1,25 +1,51 @@
-import { List } from '@material-ui/core';
 import { useIntl } from 'react-intl';
+import { Card, Divider, List } from '@material-ui/core';
 
 import TaskListItem from './TaskListItem';
 import { ZetkinTask } from '../../../../types/zetkin';
+import getTaskStatus, { TASK_STATUS } from '../../../../utils/getTaskStatus';
+
+const TASK_STATUS_ORDER: {[key in TASK_STATUS]: number} = {
+    [TASK_STATUS.DRAFT]: 0,
+    [TASK_STATUS.ACTIVE]: 1,
+    [TASK_STATUS.SCHEDULED]: 2,
+    [TASK_STATUS.CLOSED]: 3,
+};
+
+const sortTasksByStatus = (firstTask: ZetkinTask, secondTask: ZetkinTask): number => {
+    const firstTaskStatus = getTaskStatus(firstTask);
+    const secondTaskStatus = getTaskStatus(secondTask);
+    return TASK_STATUS_ORDER[firstTaskStatus] - TASK_STATUS_ORDER[secondTaskStatus];
+};
 
 interface TaskListProps {
-    tasks: ZetkinTask[];
     hrefBase: string;
+    tasks: ZetkinTask[];
 }
 
-const TaskList = ({ tasks, hrefBase }: TaskListProps): JSX.Element => {
+
+const TaskList = ({ hrefBase, tasks }: TaskListProps): JSX.Element => {
     const intl = useIntl();
 
+    const tasksOrderedByStatus = tasks.sort(sortTasksByStatus);
+
```

Comment:
maybe better to spread [...tasks].sort() so that this doesn't mutate the original array?

Reply:
Or the oldschool `tasks.concat().sort()`.

Reply:
Went with the new school way. love that spread!